Description: This patch makes sure that the CFLAGS set by dpkg-buildflags
  are not overwritten by cmake.
Forwarded: doesn't make sense upstream
Author: Ruben Undheim <ruben.undheim@gmail.com>
Index: gnuais/CMakeLists.txt
===================================================================
--- gnuais.orig/CMakeLists.txt
+++ gnuais/CMakeLists.txt
@@ -7,7 +7,7 @@ CHECK_INCLUDE_FILES(alsa/asoundlib.h HAV
 CHECK_INCLUDE_FILES(mysql/mysql.h HAVE_MYSQL)
 CHECK_INCLUDE_FILES(curl/curl.h HAVE_CURL)
 CHECK_INCLUDE_FILES(pulse/pulseaudio.h HAVE_PULSEAUDIO)
-set(CMAKE_C_FLAGS "-g -Wall -O2")
+
 
 if(NOT HAVE_ALSA)
 	MESSAGE(FATAL_ERROR "The development files for ALSA are required - libasound-dev")
